garrysmod/lua/includes/util/color.lua

```diff
@@ -162,6 +208,19 @@ function COLOR:ToHWB()
 
 end
 
+--[[---------------------------------------------------------
+	Converts color to a hex string
+-----------------------------------------------------------]]
+function COLOR:ToHex()

```

Comment:
Feels like there should be an argument to omit the alpha channel. Some implementations also use #AARRGGBB format, but I think #RRGGBBAA is better and also matches web colors implementation.

Reply:
Looked it up and #RRGGBBAA is the standard ye https://www.w3.org/TR/css-color-4/#hex-notation